modules/update-mirror-repository-oc-mirror.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network-cluster.adoc
+
+:_content-type: PROCEDURE
+[id="update-mirror-repository_oc_mirror_{context}"]
+= Mirroring images using oc-mirror
+
+Using the oc-mirror OpenShift CLI (`oc`) plug-in is the recommended method for mirroring images onto a mirror registry in a fully or partially disconnected environment.
+
+.Procedure
+
+. Navigate to link:https://docs.openshift.com/container-platform/4.11/installing/disconnected_install/installing-mirroring-disconnected.html[Mirroring images for a disconnected installation using the oc-mirror plug-in].
+
+. Follow the instructions on that page up until but not including the section titled *Keeping your mirror registry content updated*.
```

Comment:
Perhaps we need a third branch:

* Super easy: folks who have already set up oc-mirror, who only need [to keep their mirror up to date][1].
* Medium: folks who need to [set up oc-mirror for the first time][2], e.g. because they are pivoting from previously using `oc adm release mirror ...`.  These are the folks who would stop before `Keeping your mirror registry content updated`.
* Tedious: folks who keep using `oc adm release mirror ...`.

[1]: https://docs.openshift.com/container-platform/4.11/installing/disconnected_install/installing-mirroring-disconnected.html#updating-mirror-registry-content